package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "changelog-maker",
-  "version": "2.3.0",
-  "description": "A git log to CHANGELOG.md tool",
+  "name": "@agilgur5/changelog-maker",
+  "version": "0.0.1",
```

Comment:
ahh, so all the tags pre-2.3.0 exist here too as it's a fork. I guess we should start at v3.0.0 then? (probably misleading to call it stable... though I doubt many others will be using this either)

Reply:
Still a good question posed here, especially if I rebase in future upstream changes. Otherwise could delete all the tags on GitHub 🤷‍♂ 

Moving this change into a different PR as there's some more work to be done prior to publishing